Foundation/NSLog.swift

```diff
@@ -41,3 +41,11 @@ public func NSLog(_ format: String, _ args: CVarArg...) {
         NSLogv(format, $0) 
     }
 }
+
+// hook for System.out.println()
+#if os(Android)
+public var debug = {
```

Comment:
This is a temporary convenience. I can rename it or put it in the NSLog class if you like.